solr/blacklight-core/conf/solrconfig.xml

```diff
@@ -428,6 +430,7 @@
       
       <!-- for user query terms in number text box -->
       <str name="qf_number">isbn_t</str>
+      <str name="qf_number">isbn_cancelled_tim</str>
```

Comment:
```suggestion
```

Reply:
Is there a reason to keep this line?

Reply:
No, like you said the number search field isn't used! Committed your suggestion